Gemfile

```diff
@@ -7,6 +7,7 @@ group :development do
   gem 'introspection', '~> 0.0.1'
   gem 'minitest'
   gem 'rake'
+  gem 'rdoc'
```

Comment:
Unless I'm mistaken, this change is orthogonal to the other changes in this commit. If so, can you split it out into a separate explanatory commit?

Reply:
Addressed in #747.